main/CTLIterator.ml

```diff
@@ -124,8 +122,7 @@ let program_of_prog_with_termination (prog: AbstractSyntax.prog) (main: Abstract
     };
     globalBlock = globalBlock;
   } in
-  let exitProperty = StringMap.add "exit" A_TRUE (StringMap.add "" A_FALSE StringMap.empty) in
-  (program, AF (Atomic exitProperty))
+  (program, AF (Atomic (A_TRUE, Some "exit")))
```

Comment:
That is correct, the way I add exit labels to the syntax tree is by inserting them at the end of the program and before each return statement. That way we can also capture programs that use 'return' before the end of the program.

In the CFG based analysis there is only one exit label at the end.